src/monster.cpp

```diff
@@ -458,12 +456,88 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
 
 std::string monster::extended_description() const
 {
-    // @todo Add tons of info here
-    std::string ret = string_format( _ ( "This is a %s" ), name().c_str() );
-    ret += "\n--\n";
+    std::ostringstream ss;
+
+    nc_color color = c_white;
+    std::string attitude = "";
```

Comment:
Explicit construction from a c-string is redundant. Simply `std::string attitude;` would be enough.